lib/vmpooler/aws_setup.rb

```
@@ -7,11 +7,11 @@ class PoolManager
     # it uses a local key to do so and then setup SSHD on the hosts to enable
     # dev and CI users to connect.
     class AwsSetup
-      ROOT_KEYS_SCRIPT = ENV['ROOT_KEYS_SCRIPT']
+      ROOT_KEYS_SCRIPT = ENV['ROOT_KEYS_SCRIPT'] || "https://raw.githubusercontent.com/puppetlabs/puppetlabs-sshkeys/master/templates/scripts/manage_root_authorized_keys"
```

Do we have to hard code this legacy keys file directly in the provider, or can we pass in that URL as a variable (at least until we completely do away with it)?

---

in ABS it was hard coded. Here its either an ENV var or fallback to the hardcoded URL